Source/Scene/ShadowMap.js

```diff
@@ -385,6 +379,40 @@ define([
         },
 
         /**
+         * Determines if a normal bias will be applied to shadows.
+         *
+         * @memberof ShadowMap.prototype
+         * @type {Boolean}
+         * @default true
+         */
+        normalOffset : {
+            get : function() {
+                return this._normalOffset;
+            },
+            set : function(value) {
+                this.dirty = this._normalOffset !== value;
+                this._normalOffset = value;
```

Comment:
Also add a test for this. It should be fine to set `normalOffset` to false and expect the `normalOffset` to be false for all the bias modes. I don't think this requires any rendering as part of the test.